src/sfctl/custom_chaos_schedule.py

```diff
@@ -0,0 +1,163 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Custom commands for the Service Fabric chaos schedule test service"""
+
+def parse_time_of_day(time_of_day):
+    """Parse a TimeOfDay from string"""
+    from azure.servicefabric.models.time_of_day import (
+        TimeOfDay
+    )
+
+    if not time_of_day:
+        return None
+
+    hour = time_of_day.get("Hour")
+    minute = time_of_day.get("Minute")
+
+    if hour is None or minute is None:
+        return None
+
+    return TimeOfDay(hour, minute)
+
+def parse_time_range(time_range):
+    """Parse a TimeRange from string"""
+    from azure.servicefabric.models.time_range import (
+        TimeRange
+    )
+
+    if time_range is None:
+        return None
+
+    start_time = parse_time_of_day(time_range.get("StartTime"))
+    end_time = parse_time_of_day(time_range.get("EndTime"))
+
+    return TimeRange(start_time, end_time)
+
+def parse_active_time_ranges(time_ranges):
+    """Parse a list of TimeRanges from string"""
+
+    if time_ranges is None:
+        return list()
+
+    parsed_times = list()
+
+    for time_range in time_ranges:
+        parsed_times.append(parse_time_range(time_range))
+
+    return parsed_times
+
+def parse_active_days(active_days):
```

Comment:
Same here. Docstring implies that active_days is a string when it is not. Specify a dictionary of strings.